scripts/initscripts/snapd.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Starts and stops a single snapd instance on this system
+Documentation=https://github.com/intelsdi-x/snap
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+User=root
+Group=root
+Type=simple
+Restart=on-failure
+ExecStart=/usr/local/bin/snapd --config=/etc/snapd/snapd.yaml
```

Comment:
Default should be fine.  Though as the config file, and other settings can be passed via Environment variables it may be a good idea to use an "EnvironmentFile" in the service definition and have that set SNAP_CONFIG_PATH

EnvironmentFile should probably be /etc/default/snap-telemetry or /etc/sysconfig/snap-telemetry